### DISCUSSION TITLE
Not automatically saved

### DISCUSSION THREAD

**Discussion by honggwang1029** [wolveix/satisfactory-server#4] [Q&A] [answered]
This is fantastic for me.
I have been using it with satisfaction from the very beginning.
however Will not be saved since yesterday.
Can I get help?

Currently the program is running. 
I am afraid that the playing will be lost when I close Docker.
Can I save it manually?

-----------------------------------------------------------------------

I solved this problem.
By re-installation.. TT

**Comment by wolveix** [comment, marked as answer]
Hey!

You could've retrieved the save by exec'ing into the container and heading to the local save directory.

But yes, the save issue was resolved by recent revisions to this repository. Sorry if you lost any progress, that sucks!
